### DISCUSSION TITLE
Error installing cfclient

### DISCUSSION THREAD

**Discussion by cweekiat** [bitcraze/discussions#1066] [Q&A] [answered]
Hi, I followed the installation instructions for crazyflie cfclient and followed the steps as per the docs. ([link](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/)). I received the following error. 

```
INFO:__main__:Disabling STL printouts
Traceback (most recent call last):
  File "/home/me/anaconda3/envs/cf/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/me/anaconda3/envs/cf/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/me/anaconda3/envs/cf/lib/python3.9/site-packages/cfclient/gui.py", line 194, in <module>
    main()
  File "/home/me/anaconda3/envs/cf/lib/python3.9/site-packages/cfclient/gui.py", line 162, in main
    from .ui.main import MainUI
  File "/home/me/anaconda3/envs/cf/lib/python3.9/site-packages/cfclient/ui/main.py", line 35, in <module>
    from cfclient.ui.tab_toolbox import TabToolbox
  File "/home/me/anaconda3/envs/cf/lib/python3.9/site-packages/cfclient/ui/tab_toolbox.py", line 34, in <module>
    from PyQt6 import QtWidgets
ImportError: /home/me/anaconda3/envs/cf/lib/python3.9/site-packages/PyQt6/QtCore.abi3.so: undefined symbol: _ZN14QReadWriteLock16destroyRecursiveEP21QReadWriteLockPrivate, version Qt_6_PRIVATE_API

```

I have tried installing `sudo apt-get install libxcb-cursor0`. 
I am running on ubuntu 20.04, python 3.9 in a conda env. Please let me know how to resolve this.

**Comment by knmcguire** [comment]
Ah that is not the error that you see when you are missing libxcb-cursor0. Seems like you are still missing a package but unclear which one. 

 In these instructions that you gave, there is this note:
>For some versions of Ubuntu 20.04 you might need to install more packes like libxcb-cursor0. Check out [QT6 package dependency list](https://doc.qt.io/qt-6/linux-requirements.html).
Have you tried looking at your installed package list to see if these are all properly updated? You can also do a sudo apt-get update to do that as well.

**Comment by knmcguire** [comment]
Just FYI, someone has noticed this on the bitcrazeVM as well https://github.com/bitcraze/crazyflie-clients-python/issues/689

**Comment by knmcguire** [comment, marked as answer]
Fyi someone figured out what is potentially wrong. Check out this ticket: https://github.com/bitcraze/crazyflie-clients-python/issues/690

**Reply by cweekiat** [reply]
This solved the issues!
